.env

```diff
@@ -0,0 +1,11 @@
+DB_LOCAL_PORT=3307
```

Comment:
.env shouldn't be committed to version control. You can leave .env.template in this project instead, with actual values removed